experiment_helpers/set_exptDefaults.m

```diff
@@ -121,13 +121,24 @@
 durcalc.bMeasureOst = 0;
 expt = set_missingField(expt,'durcalc',durcalc);
 
-%% amplitude tracking parameters
+%% amplitude tracking parameters - old, mostly used with brut
```

Comment:
We should be using these in a lot of experiments with noise masking, so I would not add this comment.